aviatrix/resource_aviatrix_firenet.go

```diff
@@ -130,6 +130,15 @@ func resourceAviatrixFireNet() *schema.Resource {
 				Optional:    true,
 				Description: "Enable TGW segmentation for egress.",
 			},
+			"egress_static_cidrs": {
+				Type:        schema.TypeSet,
```

Comment:
Be sure to verify with the backend dev that indeed ordering is not important for this attribute.

Reply:
Wanjing said the ordering doesn't matter.